satellite/admin-back-office-v1.md

```diff
@@ -0,0 +1,636 @@
+     - Last: Last state of operation variables.
+     - Operator: Email of who conducted the change.
+4. I want there to be a simple way back to the accounts table view so that they can quickly get the
+   information needed.
+
+__When changes need to happen to a project__
+
+I want to be able to view project details per account and by project ID so that we can view and
+select the details of the project we want to change.
+
+Acceptance criteria:
+- We can see the project names and IDs.
+- We can apply user agents to projects.
+- We can see if a geo fence is applied and what the name of that geo fence is.
+- We can see the project limits and project usage.
+- Authorized users can select and change:
+  - The geo fence if the project is empty.
+  - The limits of the project: egress, storage, segments, buckets.
+  - Delete the project.
+
+__When changes need to happen to a bucket__
+
+I want to be able to view bucket details per account/per project so that we can modify/add user
+agents to buckets and apply geo fence settings to buckets.
+
+Acceptance criteria:
+- We can see the bucket names per project ID.
+- We can apply user agents to buckets.
+- We can see and apply a geo fence to an empty bucket.
+- We can see usage per bucket.
+
+__When geo fence needs to be applied to an account__
+
+I want authorized users to select a geo fence setting on the account details view and select the geo
+fence region to be applied to new projects/buckets.
+
+  Acceptance criteria:
+  - Drop down selection of available geo fence regions.
+  - The existing project and buckets will NOT get the geo fence added.
+
+__When an account needs to be suspended__
+
+1. I want authorized users to see a simple "suspend" button in the user's detail view with a drop
+   down to select the suspension reason so that we distinguish the different suspension events (e.g.
+   "billing account freeze", "other account freeze").
+
+   Acceptance criteria:
+   - Button shrinks the user's limits to 0, so that they can no longer upload or download.
+   - Drop down to select the reason for suspension:
+     - a) account delinquent.
+     - b) illegal content.
+     - c) Malicious Links.
+     - d) other.
+   - Unauthorized users should not see this button or drop down.
+2. I want the account details to show that it's obvious if a user is suspended so that they can
+   easily be manually unsuspended if needed.
+
+   Acceptance criteria:
+   - The UX makes it quick to unsuspend a user once suspended
+
+__When an account needs to be unsuspended/re-activated__
+
+I want authorized users to see a simple "reactivate" button in the user's detail view.
+
+Acceptance criteria:
+- Button resumes a user's limits to what they had them set at before suspension occurred.
+- Add an optional admin note of the reason for unsuspending.
+
+__When an account needs a limit increase or decrease__
+
+I want authorized users to be able to input the allotted usage for:
```

Comment:
I'm not sure, this was copied from Google docs. The current admin API doesn't have a rate limit change.

![image](https://github.com/storj/design-docs/assets/1731633/d6cc8eda-69b7-4df9-87d3-17ddfa7e6245)


@stoweandi what do you think?
On the other hand, the current admin API doesn't even have an account limit for a number of buckets, can you tell me if that has to be kept or removed?

Reply:
Unfortunately, we cannot add the rate limits to the account level and I had to remove the number of buckets limit too because we don't have those in the current satellite version.

If in the future, we add them, then we can add them to the admin back-office.